apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/section/exception/DeleteSectionException.java

```diff
@@ -0,0 +1,18 @@
+package gov.cdc.nbs.questionbank.page.content.section.exception;
+
+public class DeleteSectionException extends RuntimeException {
+
+    private final int errorCode;
+
+    public DeleteSectionException(String message, int errorCode){
+        super(message);
+        this.errorCode = errorCode;
+
+    }
+
+    public int getErrorCode() {
+        return errorCode;
+    }
+
+}
```

Comment:
If these new custom exceptions extend either `BadRequestException` or `NotFoundException`, then the `PageBuilderExceptionHandler` will return the "message" to the user and set the appropriate status code on the response.